core/files/entrypoint_nginx.sh

```diff
@@ -148,7 +148,7 @@ EOT
 
 update_misp_data_files(){
     for DIR in $(ls /var/www/MISP/app/files.dist); do
-        if [ "$DIR" = "certs" ]; then
+        if [ "$DIR" = "certs" ] || [ "$DIR" = "img" ]  ; then
```

Comment:
Small nit, two spaces between bracket and semicolon. Should be one.

Reply:
My bad, space removed.